### PR TITLE
Update dependencies and add typescript packages

### PR DESCRIPTION
**Pull request by eXaminator**

### CODE REVIEW

File: packages/typescript-react/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "lint",
+    "config"
+  ],
+  "license": "MIT",
+  "dependencies": {
+    "eslint-config-brainbits-typescript": "^1.0.0",
+    "eslint-config-brainbits-react": "^1.0.0"
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
```
Comment (micha149) [inline packages/typescript-react/package.json]: I think these are not necessary?

File: packages/typescript-react/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        // Append 'ts' and 'tsx' extensions to Airbnb 'import/resolver' setting
+        'import/resolver': {
+            node: {
+                extensions: ['.tsx', '.jsx', ...baseSettings['import/resolver'].node.extensions],
+            },
+        },
+        // Append 'ts' and 'tsx' extensions to Airbnb 'import/extensions' setting
+        'import/extensions': [...baseSettings['import/extensions'], '.tsx'],
+    },
+    rules: {
```
Comment (micha149) [inline packages/typescript-react/index.js]: Do we enforce prop types via typescript type definitions?

File: packages/typescript-react/tests/example.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (micha149) [inline packages/typescript-react/tests/example.jsx]: I think this is copied accidentally

File: packages/typescript/package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "@typescript-eslint/parser": "^2.19.2",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-config-brainbits": "^1.0.0"
+  },
+  "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.19.2"
+  },
+  "devDependencies": {
```
Comment (micha149) [inline packages/typescript/package.json]: Are dev deps really required here?

File: packages/typescript/tests/example.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (micha149) [inline packages/typescript/tests/example.jsx]: Accidentally copied?

File: packages/typescript-react/package.json (hunk 1 above)
Comment (eXaminator) [reply packages/typescript-react/package.json]: Correct, I'll remove it.

File: packages/typescript-react/index.js (hunk 2 above)
Comment (eXaminator) [reply packages/typescript-react/index.js]: Is there a specific rule for that? I think it should be handled via the no implicit any typescript rule automatically?

File: packages/typescript-react/tests/example.jsx (hunk 3 above)
Comment (eXaminator) [reply packages/typescript-react/tests/example.jsx]: Correct, I'll remove it.

File: packages/typescript/package.json (hunk 4 above)
Comment (eXaminator) [reply packages/typescript/package.json]: Nope, I'll remove it.

File: packages/typescript/tests/example.jsx (hunk 5 above)
Comment (eXaminator) [reply packages/typescript/tests/example.jsx]: Correct, I'll remove it.

File: packages/typescript-react/index.js (hunk 2 above)
Comment (micha149) [reply packages/typescript-react/index.js]: I don't know. Thats why I'm asking 😅

**Pull request merged**